File: src/main/java/com/codingforcookies/betterrecords/item/ModItems.java
package com.codingforcookies.betterrecords.item;

import net.minecraft.item.Item;

public final class ModItems {

    public static Item itemURLRecord;
    public static Item itemURLMultiRecord;
    public static Item itemFreqCrystal;

    public static Item itemRecordWire;
    public static Item itemRecordCutters;

    public static void init() {
        itemURLRecord = new ItemRecord("urlrecord");
        itemURLMultiRecord = new ItemMultiRecord("urlmultirecord");
        itemFreqCrystal = new ItemFrequencyCrystal("freqcrystal");
        itemRecordWire = new ItemWire("recordwire");
        itemRecordCutters = new ItemWireCutter("recordwirecutters");
    }
}

File: src/main/kotlin/com/codingforcookies/betterrecords/item/ModItems.kt
package com.codingforcookies.betterrecords.item

import com.codingforcookies.betterrecords.ID
import net.minecraft.item.Item
import net.minecraftforge.event.RegistryEvent
import net.minecraftforge.fml.common.Mod
import net.minecraftforge.fml.common.eventhandler.SubscribeEvent

@Mod.EventBusSubscriber(modid = ID)
object ModItems {

    val itemRecord: ModItem = ItemRecord("record")
    val itemMultiRecord: ModItem = ItemMultiRecord("multirecord")
    val itemFrequencyCrystal: ModItem = ItemFrequencyCrystal("frequencycrystal")
    val itemWire: ModItem = ItemWire("wire")
    val itemWireCutters: ModItem = ItemWireCutter("wirecutters")

    @SubscribeEvent
    fun registerItems(event: RegistryEvent.Register<Item>) {
        event.registry.registerAll(
                itemRecord,
                itemMultiRecord,
                itemFrequencyCrystal,
                itemWire,
                itemWireCutters
        )
    }
}
